compiler/frontend/src/string_to_rcst/text.rs

```diff
@@ -16,8 +16,17 @@ use tracing::instrument;
 pub fn text(input: &str, indentation: usize) -> Option<(&str, Rcst)> {
     let (input, opening_single_quotes) = parse_multiple(input, single_quote, None)?;
     let (mut input, opening_double_quote) = double_quote(input)?;
-    let (new_input, opening_whitespace) = whitespaces_and_newlines(input, indentation + 1, false);
-    input = new_input;
+    let (new_input, mut opening_whitespace) =
+        whitespaces_and_newlines(input, indentation + 1, false);
+
```

Comment:
I'd move the blank line above because this and the `if`/`else` below belong together

```suggestion

    let (new_input, mut opening_whitespace) =
        whitespaces_and_newlines(input, indentation + 1, false);
```